src/app/container/search/search.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, ElementRef, EventEmitter, Output, ViewChild } from '@angular/core';
+
+@Component({
+  selector: 'app-search',
+  templateUrl: './search.component.html',
+  styleUrls: ['./search.component.css']
+})
+export class SearchComponent {
+  searchText: string = ''
+
+  @Output()
+  updatedSearchText: EventEmitter<string> = new EventEmitter<string>()
+
+  @ViewChild('searchInput') searchInput: ElementRef
```

Comment:
You should not use @ViewChild just to grab the html value of the input, you should learn how to use ReactiveForms and how they help to control inputs